exercises/armstrong-numbers/canonical-data.json

```diff
@@ -0,0 +1,86 @@
+{
+  "exercise": "armstrong-number",
+  "version": "1.0.0",
+  "cases": [
+    {
+      "description": "0 is an Armstrong number",
+      "property": "armstrong",
```

Comment:
For boolean options the preferred property would be: "isArmstrongNumber"

Reply:
Fixed